exdb/tests/integration_tests.py

```diff
@@ -88,94 +88,105 @@ def setUp(self):
         self.test_org = self.create_org()
         self.test_keyword = self.create_keyword()
 
+    def get_post_data(self, start, end, name='test', description='test', ex_type=None, sub_type=None, audience='c',
+                      guest='1', recognition=None, keywords=None, goal='a'):
+        if not ex_type:
+            ex_type = self.test_type
+        if not sub_type:
+            sub_type = self.test_sub_type
+        if not recognition:
+            recognition = [self.test_org.pk]
+        if not keywords:
+            keywords = [self.test_keyword.pk]
```

Comment:
These could all be in the format:
keywords = keywords or [self.test_keyword.pk]

but that's just nit-picking.